### PR TITLE
7 feature7 TitleComponent

### LINKED ISSUE #7: feature#7-TitleComponent
# REQUIREMENTS
Implements a standard title component to use on all the screens.

The component should receive a text parameter to change the title and refactors previous screens using this component

# DESIGN

imagen

### PR DESCRIPTION
**Pull request by Astroyayo**
# Description
* Creates a standard component for titles that receive a text prop to assign the title
* Refactors preview screen (SignUpScreen) to use this component
  
  ## Screenshots 
image

### CODE REVIEW

File: flight-reservation/components/TitleComponent/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { Text } from "react-native";
+import { styles } from "./styles";
+
+const TitleComponent = ({title}) => {
+    return(
+        <Text style={styles.title}>{title}</Text>
+    )
+
```
Comment (wiltrouble) [inline flight-reservation/components/TitleComponent/index.js]: remove this empty row

File: flight-reservation/components/index.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import ButtonComponent from './ButtonComponent'
 import CheckBoxComponent from './CheckBoxComponent'
 import TextInputComponent from './TextInputComponent';
+import TitleComponent from './TitleComponent';
 
-export {ButtonComponent, CheckBoxComponent, TextInputComponent}
+export {ButtonComponent, CheckBoxComponent, TextInputComponent,TitleComponent}
```
Comment (wiltrouble) [inline flight-reservation/components/index.js]: add a space after the comma

File: flight-reservation/components/TitleComponent/index.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import { styles } from "./styles";
+
+const TitleComponent = ({title}) => {
+    return(
+        <Text style={styles.title}>{title}</Text>
+    )
+    
+}
+
+export default TitleComponent
```
Comment (wiltrouble) [inline flight-reservation/components/TitleComponent/index.js]: add a row after the export

**Review by wiltrouble — CHANGES_REQUESTED** [review]

**Review by TonyHdez-Pi — APPROVED** [review]

**Review by featlast — APPROVED** [review]
well done (fix return)

File: flight-reservation/components/TitleComponent/index.js (hunk 1 above)
Comment (Astroyayo) [reply flight-reservation/components/TitleComponent/index.js]: addressed

File: flight-reservation/components/index.js (hunk 2 above)
Comment (Astroyayo) [reply flight-reservation/components/index.js]: addressed

File: flight-reservation/components/TitleComponent/index.js (hunk 3 above)
Comment (Astroyayo) [reply flight-reservation/components/TitleComponent/index.js]: addressed

**Review by wiltrouble — APPROVED** [review]
Looks good

**Pull request merged**